### ISSUE TITLE
ng_controller_name not working with regexp in config file

### ISSUE THREAD

**Issue by hindsholm** [Gillespie59/eslint-plugin-angular#81]
When ng_controller_name is configured from a configuration file, a regex cannot be specified.

To verify, check out demo/gulp and change the controller name to `AppController`, which matches the regex `'/[A-Z].*Controller$/'` in .eslintrc. The result is:

```bash
$ gulp
4:1   error  The AppController controller should be prefixed by /[A-Z].*Controller$/   angular/ng_controller_name
```

The error message indicates that the regex is being interpreted as a string, and indeed utils.js contains this test:

```javascript
isRegexp: function(regexp){
    return toString.call(regexp) === '[object RegExp]';
}
```

But I guess since the regex in this case is read from .eslintrc as a string, it will never be interpreted as a regex.

I have not supplied a pull request, because I don't know, how you want to solve this. My suggestion is to only support regex matching because it can easily be used for "begins with" as well.

And BTW: Thanx for a great plugin!

**Comment by Gillespie59** [comment]
Hi @hindsholm 
Thanks for this report. This issue has been solved by @dustinspecker in this commit https://github.com/Gillespie59/eslint-plugin-angular/commit/80336f1b844c951852b3131c0ff072f301e31a95
This changes will be available in the next 0.0.7 release. You can test this version hosted on the master branch, it will solve your issue I hope.

**Comment by hindsholm** [comment]
Nice to see, that this has already been fixed.
I will have a look at master or wait for the 0.0.7 release.

**Comment by Gillespie59** [comment]
This issue is solved, and available in the new 0.0.7 branch.
